x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -746,7 +748,9 @@ export class RulesClient {
       result.attributes.alertTypeId,
       result.attributes,
       result.references,
-      includeLegacyId
+      includeLegacyId,
+      false,
+      includeSnoozeData
```

Comment:
Im wondering if this would be a breaking change for the `/_resolve` endpoint since the response will have new data. 

Although the `/_resolve` route is internal and is only used by the rule details page (from what I can tell). Security solution does use the `rulesClient.resolve` function but they call it directly in code, which means they don't have to pass in the `includeSnoozeData` prop.

Reply:
if the route is internal, we are fine!